nautobot/core/views/routers.py

```diff
@@ -0,0 +1,82 @@
+from rest_framework.routers import Route, SimpleRouter
+
+
+class NautobotUIViewSetRouter(SimpleRouter):
+    """
+    Django Rest Framework ViewSet Custom Router.
```

Comment:
Please revise to mention that this is intended for use for the UI. Eeven though it's in the name, the docstring should vibe.